components/google_sheets/google_sheets.app.mjs

```diff
@@ -102,7 +102,15 @@ export default {
           return [];
         }
 
-        return value.replace(/["'[\]\s]+/g, "").split(",");
+        if (value[0] === "[") {
+          value = value.substr(1, value.length);
+        }
+
+        if (value[value.length - 1] === "]") {
+          value = value.substr(0, value.length - 1);
+        }
+
+        return value.replace(/["']+/g, "").split(",");
```

Comment:
This looks good and should allow whitespace within string elements in the array.

Reply:
To allow spaces between array elements and commas/brackets (e.g. `[ "foo", "bar" ]`), I think we could split the string on "commas surrounded by whitespace" instead of just commas. For example:
```js
        return value
          // Remove square brackets from ends ([ "foo", 5 ] ->  "foo", 5 )
          .replace(/(^\[)|(]$)/g, "")
          .trim() // ( "foo", 5  -> "foo", 5)
          // Remove quotes from ends ("foo", 5  ->  foo", 5)
          .replace(/^["']|["']$/g, "")
          // Split on quotes, whitespace, and comma (foo", 5 ->  ["foo","5"])
          .split(/["']?\s*,\s*["']?/);
```

Reply:
Let me do some tests

Reply:
Yeah, looks pretty good, I did some tests, and looks great